src/main/java/bio/terra/service/snapshotbuilder/utils/constants/ConceptAncestor.java

```diff
@@ -6,4 +6,5 @@ private ConceptAncestor() {}
   public static final String TABLE_NAME = "concept_ancestor";
   public static final String ANCESTOR_CONCEPT_ID = "ancestor_concept_id";
   public static final String DESCENDANT_CONCEPT_ID = "descendant_concept_id";
+  public static final String MIN_LEVELS_OF_SEPARATION = "min_levels_of_separation";
```

Comment:
Do we still need this?

Reply:
Yes, it is still used to calculate has_children (As a filter so that we don't have to join on so many records)